poseidonExt_test.go

```diff
@@ -0,0 +1,38 @@
+package circuits
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"testing"
+)
+
+func TestPoseidonHash_64input(t *testing.T) {
```

Comment:
In addition, better to cover this edge cases for number of elements equal to:

-  <5 
- =5
- 5 * n (just one case when multiple of 5)

* I treat that you hash by 5 elements (not by 6) in each round